client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -238,6 +258,19 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		}
 	}
 
+	function recordStepContainerTracksEvent( eventName: string ) {
+		const tracksProps = {
+			step: showGeneratedDesigns ? 'generated-design-step' : 'design-step',
+			intent: intent,
+		};
+
+		if ( isForceStaticDesigns ) {
```

Comment:
It looks like this is also being called when you exit the preview for a static design (using the back button)

Reply:
Nice catch!